### PR TITLE
imporoving vb4 importer

### PR DESCRIPTION
**Pull request by mabras**

### CODE REVIEW

File: script/import_scripts/vbulletin.rb [hunk 1]
```diff
@@ -469,15 +473,15 @@ def close_topics
     sql = <<-SQL
       WITH closed_topic_ids AS (
         SELECT t.id AS topic_id
-        FROM #{TABLE_PREFIX}post_custom_fields pcf
```
Comment (techAPJ) [inline script/import_scripts/vbulletin.rb]: Why are you removing `#{TABLE_PREFIX}` here (and in 3 more places below)?

File: script/import_scripts/vbulletin.rb [hunk 2]
```diff
@@ -718,22 +722,20 @@ def suspend_users
...
-        user.suspended_at = parse_timestamp(user["bandate"])
-        user.suspended_till = 200.years.from_now
-
-        if user.save
-          StaffActionLogger.new(system_user).log_user_suspend(user, "banned during initial import")
-          banned += 1
-        else
-          puts "Failed to suspend user #{user.username}. #{user.errors.try(:full_messages).try(:inspect)}"
-          failed += 1
+    User.find_each do |user|
```
Comment (techAPJ) [inline script/import_scripts/vbulletin.rb]: Looping through all the users does not seem like a better approach here.

I know that current `suspend_users` method is broken and I have a (single line) fix for this in place which I will commit soon. Can you please omit this change?

File: script/import_scripts/vbulletin.rb (hunk 1 above)
Comment (mabras) [inline script/import_scripts/vbulletin.rb]: This query is for postgresql database (Discourse database) not for mysql (VB4 database).

Comment (techAPJ) [inline script/import_scripts/vbulletin.rb]: Aha, I see now! 👍

File: script/import_scripts/vbulletin.rb (hunk 2 above)
Comment (mabras) [inline script/import_scripts/vbulletin.rb]: Ok, I will omit this.

**Comment by mabras** [comment]
I have omitted the changes you asked for.

I just want to ask you, is okay to replace all the `gsub` method with `gsub!` to save to memory?

**Comment by techAPJ** [comment]
Sure, go ahead! :+1:

**Comment by mabras** [comment]
I did replace the `gsub` method. 

In addition to that the `@closed_topic_ids` always return an empty array duo to that `topic["open"]` return `Fixnum` object and we `==` against `String` object.  This broke the `close_topics` method.

**Comment by techAPJ** [comment]
Thanks!

**Pull request merged**